src/CustomMySQL.js

```diff
@@ -115,7 +116,19 @@ export default class CustomMySQL {
     }
 
     build () {
-        this._cache = Array.from(arguments);
+
+        const args = Array.from(arguments);
+
+        if (args.length === 1 && squel.cls.isSquelBuilder(args[0])) {
+
+            const query = args[0].toParam();
+
+            this._cache = [query.text, query.values];
```

Comment:
Why not keep L118 and just do 

this._cache[0] = query.text;
this._cache[0].splice(1, 0, query.values);